lib/scala/text-buffer/src/main/scala/org/enso/text/Sha3_224VersionCalculator.scala

```diff
@@ -2,14 +2,14 @@ package org.enso.text
 
 import java.nio.charset.StandardCharsets
 
-import org.bouncycastle.jcajce.provider.digest.SHA3
+import java.security.MessageDigest
 
 /** SHA3-224 digest calculator. */
 object Sha3_224VersionCalculator extends ContentBasedVersioning {
 
   /** @inheritdoc */
   override def evalVersion(content: CharSequence): ContentVersion = {
-    val digestSHA3 = new SHA3.Digest224()
+    val digestSHA3 = MessageDigest.getInstance("SHA3-224")
```

Comment:
Use SHA1, if possible.

Reply:
Tracked as a separate issue in https://github.com/enso-org/enso/issues/8662